docs/sql-guide/expressions/expressions-home.md

```diff
@@ -1,42 +1,80 @@
 ---
-title: SQL Expressions
+title: Expressions
 layout: default
 parent: SQL guide
-has_children: true
-nav_order: 5
-has_toc: false
 ---
 
 # SQL Expressions
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-db-create-before-begin.md %}
 
 ## identifier
 ![expr](/assets/images/sql-guide/identifier.svg)
 
+{% include /concepts/standard-naming-obj.md %}
+
 ## expr
+
 ![expr](/assets/images/sql-guide/expr.svg)
 
-## integer_literal
+| Syntax | Example | Result |
+|---|---|---|
+| `expr [NOT] LIKE expr` | `SELECT * from Products where product_name LIKE pen` | `ballpen`, `pencil`, `playpen` |
+| `expr IS [NOT] null` | `SELECT * from Services where Price is null` | `free delivery on orders over $50` |
+| `expr [NOT] between expr AND expr` | `SELECT * from Products WHERE Price BETWEEN 10 AND 20;` | `fountain pen`, `notepad`, `rubber ball` |
+| `expr [NOT] IN ([SELECT_stmnt | expr,...])` | `Select * from Products where product_ID NOT IN (SELECT * from Sales where product_name LIKE ear)` | `gold-plated earplugs` |
```

Comment:
Um, I cheated and didn't test. But I can create the tables and give it a whirl ;D

Reply:
fixed the sql as suggested, but as I say, I'll create the table and give this a test to see what comes out the other end :D

Reply:
I'm massaging the SQL examples right now and SLACKed you some issues I've run into with statements that should in theory work. Unless I've munted them because of being an idiot ;)

Reply:
Also I confess to having *no idea* what I was getting at when I wrote `Select * from Products where product_ID NOT IN (SELECT product_ID from Sales where product_name LIKE ear)` and I've had trouble working out a SQL statement that's meaningful. Care to offer some SQL? <smiles encouragingly> :)

Reply:
Let's get this PR in and stop adding. We can add examples later, and I can help. If the sql runs without a bug, keep it